src/features/Main/Sidebar/index.jsx

```diff
@@ -1,91 +1,71 @@
 import React from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { useHistory } from 'react-router-dom';
+
+import { Sidebar as SidebarBase, MenuSection, MenuItem } from 'react-paragon-topaz';
+
 import { updateActiveTab } from 'features/Main/data/slice';
 import './index.scss';
 
+const menuItems = [
+  {
+    link: 'dashboard',
+    label: 'Home',
+    icon: <i className="fa-regular fa-house" />,
+  },
+  {
+    link: 'licenses',
+    label: 'License inventory',
```

Comment:
License inventory doesn't fit in one line when is active 
![Screenshot from 2024-05-23 14-31-12](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/36944773/c0c93cdd-8ede-477e-8c88-1bf7d3f0c4c3)